src/EditorFeatures/VisualBasicTest/Diagnostics/Configuration/ConfigureSeverity/CodeStyleOptionBasedSeverityConfigurationTests.vb

```diff
@@ -253,7 +253,7 @@ Class Program1
 End Class
         </Document>
         <AnalyzerConfigDocument FilePath=""z:\\.editorconfig"">[*.{cs,vb}]
-dotnet_style_object_initializer = false:suggestion
+dotnet_style_object_initializer = true:suggestion
```

Comment:
❓ Why was this change made?

Reply:
nm, I should have read https://github.com/dotnet/roslyn/commit/559c17ae0baceb4e45ca4c82ac50743a9dc95dac